content/it/container-orchestration.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Container Orchestration
+status: Completed
+category: Concetto
+---
+
+## Cos'è
+Per orchestrazione dei [Container](/it/container/) si intende la gestione e l'automazione del ciclo di vita delle applicazioni containerizzate in ambienti dinamici.
+L'orchestrazione dei container viene effettuata tramite l'utilizzo di un orchestratore di container (nella maggior parte dei casi [Kubernetes](/it/kubernetes)), che consente di eseguire i processi di deployement, (auto)scaling, auto-healing e monitoraggio.
```

Comment:
In una riunione moooolto vecchia (vedi [qui](https://docs.google.com/document/d/1g4__gy--tB79UH_KMcWFNbJpPQQUhlVpqnKKGGndOZU/edit#bookmark=id.bxndpdwsldp0)), si era deciso di lasciare i termini inglesi, dove si ritenesse necessario, ma con una traduzione in italiano tra parentesi. Qui secondo me è utile tenere i termini inglesi, ma anche una traduzione in Italiano tra parentesi credo farebbe bene.  che ne dici
```suggestion
L'orchestrazione dei container viene effettuata tramite l'utilizzo di un orchestratore di container (nella maggior parte dei casi [Kubernetes](/it/kubernetes)), che consente di eseguire i processi di deployment (distribuzione), (auto)scaling (scalabilità), auto-healing (riparazione) e monitoraggio.
```